src/plugins/discover/public/components/discover_grid/get_render_cell_value.tsx

```diff
@@ -163,20 +108,6 @@ export const getRenderCellValueFn =
       );
     }
 
-    if (!field?.type && rowFlattened && typeof rowFlattened[columnId] === 'object') {
-      if (isDetails) {
-        // nicely formatted JSON for the expanded view
-        return (
-          <JsonCodeEditor
-            json={rowFlattened[columnId] as Record<string, unknown>}
-            width={defaultMonacoEditorWidth}
-          />
-        );
-      }
-
-      return <>{formatFieldValue(rowFlattened[columnId], row, indexPattern, field)}</>;
```

Comment:
So basically, this case will be rendered using span below, right?

Reply:
yes, I'm simplifying code here